manifests/charts/istiod-remote/files/gen-istiod-remote.yaml

```diff
@@ -674,6 +674,7 @@ data:
           securityContext:
             fsGroup: 1337
           {{- end }}
+      gateway: |
```

Comment:
Doesn't seem right - shouldn't it have the template here as well ?

Reply:
nice catch, bug in the makefile